scm-ui/ui-components/src/layout/Page.tsx

```diff
@@ -33,6 +33,7 @@ import ErrorBoundary from "../ErrorBoundary";
 
 type Props = {
   title?: string;
+  titleComponent?: ReactNode;
```

Comment:
I'm not sure if i like this change. Could we not use ReactNode as `title` (ReactNode includes string, so it is not a breaking change) and pass the title prop always as child to the Title component?

Reply:
If title is something different than a string, I need to have a dedicated string title for the browser, here. Would you prefer to have something like a 'textTitle' that can be set, when title as a ReactNode? In this case it would not be transparent, that you have to set it in some cases. Keeping the title a string and offering an additional component prop solves this in my view, so I chose this solution.

Reply:
Collectively we decided to do as @sdorra suggested and add an additional property for the document title.